nominal/nominal.py

```diff
@@ -653,6 +653,22 @@ def create_event(
     return conn.create_event(name, type, start, duration, asset_rids=asset_rids, properties=properties, labels=labels)
 
 
+def search_events(
+    *,
+    search_text: str | None = None,
+    after: datetime | ts.IntegralNanosecondsUTC | None = None,
+    before: datetime | ts.IntegralNanosecondsUTC | None = None,
```

Comment:
nit but for parity with other functions, the type could be `str | datetime | ...`